### ISSUE TITLE
type: integer shown as "any" on schemas section

### ISSUE THREAD

**Issue by criess** [swagger-api/swagger-ui#8959]
### Q&A (please complete the following information)
 - OS: Fedora 38
 - Browser: FF
 - Version: 114.0.2 (64-Bit)
 - Method of installation: dist folder from https://github.com/swagger-api/swagger-ui/releases/tag/v5.1.0
 - Swagger-UI version: 5.1.0
 - Swagger/OpenAPI version: Openapi 3.1.0 (see example)

### Content & configuration

Just use attached example with dist(ributed) folder from releases

Example Swagger/OpenAPI definition:
```yaml
openapi: 3.1.0
info:
  title: "example `type: integer` on array-wrapped objects"
components:
  schemas:
    my_schema_with_keys_that_contain_arry:
      type: object
      properties:
        outside_array:
          type: array
          items:
            type: object
            properties:
              id:
                type: integer
              id_or_null:
                type:
                  - integer
                  - "null"

```

Swagger-UI configuration options:
```js
  window.ui = SwaggerUIBundle({
    urls: [{name: 'DataLake v1',  url: './data_lake_v1.yaml'},{name: 'API v3', url: './api_v3.yaml'}],
    dom_id: '#swagger-ui',
    deepLinking: true,
    presets: [
      SwaggerUIBundle.presets.apis,
      SwaggerUIStandalonePreset
    ],
    plugins: [
      SwaggerUIBundle.plugins.DownloadUrl
    ],
    layout: "StandaloneLayout"
  });
```

api_v3.yaml holds example above!

```
just opened index.html without any query!
```

### Describe the bug you're encountering

integer type for id field shown as `any`

### To reproduce

load above provided schema and check rendering of schemas it would indicate `any` instead of `integer` for field id (from an object inside array though!) 

### Expected behavior

expect to see integer (without null) just liked on mixed type `id_or_null`

### Screenshots

![grafik](https://github.com/swagger-api/swagger-ui/assets/1207146/c468fad1-2ac6-4907-b59e-550fac132171)

**Comment by calidan-x** [comment]
<img width="1219" alt="Screenshot 2023-06-28 at 11 18 36" src="https://github.com/swagger-api/swagger-ui/assets/57343735/a24d3e27-e1e1-4547-8314-8790d957ab14">

**Comment by srka99** [comment]
For the fix, I believe "integer" should be included here;
https://github.com/swagger-api/swagger-ui/blob/master/src/core/plugins/json-schema-2020-12/fn.js#L120

**Comment by glowcloud** [comment]
Closing the issue, as this was fixed with the [v5.1.3](https://github.com/swagger-api/swagger-ui/releases/tag/v5.1.3) release.
